adapter/noop2/noop.go

```diff
@@ -124,31 +123,31 @@ func (o *obuilder) Build(cfg adapter.Config, env adapter.Env) (adapter.Handler,
 }
 
 // ConfigureCheckNothingHandler is to be deleted
-func (*obuilder) ConfigureCheckNothingHandler(map[string]*checknothing.Type) error {
+func (*obuilder) SetCheckNothingTypes(map[string]*checknothing.Type) error {
 	return nil
 }
 
 // ConfigureReportNothingHandler is to be deleted
-func (*obuilder) ConfigureReportNothingHandler(map[string]*reportnothing.Type) error {
+func (*obuilder) SetReportNothingTypes(map[string]*reportnothing.Type) error {
```

Comment:
Of course, you need to change all these useless comments too for the linters to be happy...

Reply:
Oho ya..Surprising Linter is passing. Will delete this code in the next PR.